climpred_skeleton/alignment.py

```diff
@@ -0,0 +1,21 @@
+import xarray as xr
+
+from .comparison import Comparison
+
+
+class LeadAlignment(Comparison):
+    def __init__(
+        self,
+        prediction: xr.Dataset,
+        observation: xr.Dataset,
+        compare: str,
+        alignment: str,
+    ):
+        super().__init__(prediction, observation)
```

Comment:
Maybe I should actually not have this inherit from `Comparison` but instead just instantiate a `Comparison` and run `factory()` so we don't inherit `.broadcast()` and `.factory()`. They should be run once to grab the correct comparison and then broadcast things properly.